crates/rover-client/src/query/config/whoami.rs

```diff
@@ -48,14 +48,21 @@ fn get_identity_from_response_data(
         // graphs as api key actors, since that's all we _should_ get.
         // I think it's safe to only include those two kinds of actors in the enum
         // more here: https://studio-staging.apollographql.com/graph/engine/schema/reference/enums/ActorType?variant=prod
+
         let key_actor_type = match me.as_actor.type_ {
             who_am_i_query::ActorType::GRAPH => Actor::GRAPH,
             who_am_i_query::ActorType::USER => Actor::USER,
             _ => Actor::OTHER,
         };
+
+        let title = match me.on {
+            who_am_i_query::WhoAmIQueryMeOn::Service(s) => s.title,
+            _ => "No Title".to_string(),
```

Comment:
Happy to have `No Title` as something else, if that doesn't sound suitable.

Reply:
I doubt we'll see this case often (if ever?), but regardless, I think this is fine :)

Reply:
Ah, I see what's going on here. It's printing `service title` regardless of whether it's a service key or not. I think I'm going to push a commit changing that, feel free to reject it or push back tomorrow :)

Reply:
Ah, yea this new commit you pushed works well. Merging the PR in.